les/redeemed-winnercheque-v1.jsonld

```diff
@@ -0,0 +1,18 @@
+{
+   "@context":{
+      "@version": 1.1,
+      "wc": "https://sphereon-opensource.github.io/vc-contexts/les/winnercheque-v1#",
+      "schema": "http://schema.org/",
+      "RedeemedWinnerCheque": {
+         "@id": "https://sphereon-opensource.github.io/vc-contexts/les/redeemed-winnercheque-v1#rwc",
+         "@context": {
+            "@version": 1.1,
+            "@protected": true,
+            "id": "@id",
+            "type": "@type",
+            "redeemedDate": "schema:DateTime",
+            "winnerCheque": "wc:LESWinnerCheque"
+         }
```

Comment:
I would expect a transactionId in here. Just before issueance of this VC a Coinbase transaction has happened from the player address (managed by the Casino for now), back to the Casino. I would expect the transactionId to be present, so that anyone inspecting the VC can verify the transaction happened

Reply:
added

Reply:
We also have a coinbase transaction from casino to player during player wins and leaves the casino(During winner cheque creation). 
Should we also have transactionId inside VerifiableCredential for playerLeavesCasino use case when we first created the winner cheque?

Reply:
Yes. Since we now moved all of the VC issuance to the back, meaning they are dependent on successful transactions. It also means we should include that type of data in the VCs as proofs

Reply:
@mehmetbicak Marking this one as resolved for this context, but indeed should be added to the other context

Reply:
I created another ticket for this. 
https://sphereon.atlassian.net/browse/LES-68